src/interpolation/interpolate.test.js

```diff
@@ -97,23 +121,148 @@ describe('findPlaceholders', () => {
     }
     const placeholders = findPlaceholders(document)
 
-    expect(placeholders).toEqual([' user.name ', 'movie.title | uppercase', ' movie.rating '])
+    expect(placeholders).toMatchSnapshot()
   })
 })
 
-describe('computeUpdates', () => {
+describe('analyzePlaceholders', () => {
   test('no placeholders', () => {
-    const updates = computeUpdates([], {}, {})
+    const placeholders = analyzePlaceholders([], {})
+
+    expect(placeholders).toEqual([])
+  })
+
+  test('contentPlaceholder - text', () => {
+    const placeholders = [
+      {
+        raw: '{{ user.name.first | lowercase }}',
+        position: {
+          start: 134,
+          end: 166
+        }
+      }
+    ]
+
+    const contentPlaceholders = analyzePlaceholders(placeholders, {
+      user: { name: { first: 'Thibaud' } }
+    })
+
+    expect(placeholders).toMatchSnapshot()
```

Comment:
should be `contentPlaceholders` that is expected to match the snap, right?

Snap should then be
```js
exports[`analyzePlaceholders contentPlaceholder - text 1`] = `
Array [
  Object {
    "input": Object {
      "raw": "user.name.first",
      "value": "Thibaud",
    },
    "output": "thibaud",
    "pipes": Array [
      Object {
        "output": "thibaud",
        "raw": "lowercase",
      },
    ],
    "position": Object {
      "end": 166,
      "start": 134,
    },
    "raw": "{{ user.name.first | lowercase }}",
    "type": "content",
  },
]
`;
```

Reply:
Oh yes, that's a mistake